### PR TITLE
Redirect Users to GitHub's Releases Page

### PR DESCRIPTION
**Pull request by RadWolfie**
Plus remove AppVeyor and Travis badges since they are currently disabled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +17,11 @@ Cxbx-Reloaded is an emulator for running Microsoft Xbox (and eventually, Chihiro
     * Make sure to enable winpcap compatibility mode!
 
 ## Automated Builds
-Cxbx-Reloaded is not yet ready for general release, but the latest development builds can be downloaded using the links below:
+Cxbx-Reloaded does not currently have stable builds, but you can go to the Releases tab to obtain pre-release builds or use the links below:
```
Comment (Margen67) [inline README.md:20]:
```suggestion
Cxbx-Reloaded doesn't currently have stable builds, but you can obtain pre-release builds from the Releases tab, or the links below:
```

File: README.md [hunk 2]
```diff
@@ -19,11 +17,11 @@ Cxbx-Reloaded is an emulator for running Microsoft Xbox (and eventually, Chihiro
...
 
-  * **[Latest Development Build](https://ci.appveyor.com/api/projects/SoullessSentinel/cxbx-reloaded/artifacts/Release.zip?branch=develop&job=Configuration:%20Release&pr=false)**
-
-  * *[Full build history](https://ci.appveyor.com/project/SoullessSentinel/cxbx-reloaded/history)*
+  * **[Release Builds](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/releases)**
+    * **WINE users will need to use `CxbxReloaded-Release-VS2017.zip` for it to run.**
```
Comment (Margen67) [inline README.md:23]: You can also word it like this:
```suggestion
    * **WINE users will need to use `CxbxReloaded-Release-VS2017.zip` for it to run correctly.**
```

File: README.md (hunk 1 above)
Comment (RadWolfie) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (RadWolfie) [reply README.md]: fixed

**Pull request merged**